masala-parser.d.ts

```diff
@@ -247,7 +245,7 @@ export interface Stream<Data> {
 interface Streams {
     ofString(string: string): Stream<string>;
 
-    ofArray<X>(): Stream<Array<X>>;
+    ofArray<X>(): Stream<X[]>;
```

Comment:
`Stream<Array<X>` is preferred to `Stream<X[ ]>` . It's more natural to read when two <> are involved.